lib/core/macros.nim

```diff
@@ -1435,9 +1435,9 @@ proc hasArgOfName*(params: NimNode; name: string): bool {.compileTime.}=
   ## Search ``nnkFormalParams`` for an argument.
   expectKind(params, nnkFormalParams)
   for i in 1 ..< params.len:
-    template node: untyped = params[i]
-    if name.eqIdent( $ node[0]):
-      return true
+    for j in 0..<params[i].len-2:
```

Comment:
need space

Reply:
No?

Reply:
The code above use space, maybe keep consistent:
```nim
for i in 1 ..< params.len:
```

Reply:
Its more common without spaces and looks better so I'm gonna change the other line too.